src/MakeListing/buylisting.js

```diff
@@ -28,7 +36,26 @@ const BuyListing = () => {
     return (
         <Styles>
             <div className="">
-                <NavigationBar/>
+                <div className="row mt-5">
+                    <div className="col-md-6 offset-md-3 text-center">
+                        <div className="progress">
+                            <div className="progress-bar" role="progressbar"
+                                 aria-valuenow="25"
+                                 aria-valuemin="0" aria-valuemax="100"></div>
+                        </div>
+                        <div className="row">
+                            <div className="col-md-4 breadcrumb-text">
+                                Create a Listing
+                            </div>
+                            <div className="col-md-4 breadcrumb-text">
+                                Subscription
+                            </div>
+                            <div className="col-md-4 breadcrumb-text">
+                                Checkout
+                            </div>
+                        </div>
+                    </div>
+                </div>
```

Comment:
I was confused about this a little. since the page before buylisting is makelisting, if listingData has to come from there then theres no information of things like Amount Due Today on makelisting. nevertheless I;ve made an object listingData in makelistings and when create is clicked it send that to buylistings and then the data is populated.